### ISSUE TITLE
Implement shapeless recipes

### ISSUE THREAD

**Issue by SirCmpwn** [SirCmpwn/TrueCraft#28]
i.e. torches

**Comment by Mechazawa** [comment]
I can't think of any shapeless recipes from the top of my head (dyed wool?) but if I recall correctly torches require the coal to be above the stick. 

![](http://oyster.ignimgs.com/mediawiki/apis.ign.com/minecraft/6/6c/Torch_recipe.jpg)

**Comment by SirCmpwn** [comment]
You're right - torches became shapeless in a later update. Dyed wool is a better example.

**Comment by brudin** [comment]
Mushroom stew is shapeless, same goes with flint and steel, and probably some others.
